pallets/attestation/src/tests.rs

```diff
@@ -183,15 +183,25 @@ fn test_revoke_remove() {
 				Attestation::attestations(claim_hash).expect("Attestation should be present on chain.");
 
 			assert!(stored_attestation.revoked);
-			assert_eq!(Balances::reserved_balance(ACCOUNT_00), <Test as Config>::Deposit::get());
+			assert_eq!(
+				Balances::balance_on_hold(
+					&<Test as Config>::RuntimeHoldReason::from(HoldReason::Deposit),
```

Comment:
Easier to do `(HoldReason::Deposit.into()` (everywhere else). Inside and outside this module.

Reply:
Resolved: [2ae8611](https://github.com/KILTprotocol/kilt-node/pull/527/commits/2ae86115b5d4a963e9eb232236e94971fb0d9bb6)